packages/flutter_test/lib/src/window.dart

```diff
@@ -2,9 +2,13 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-import 'dart:ui' as ui hide window;
+import 'dart:ui' hide window;
 
 import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart' show MaterialApp;
```

Comment:
This import is ... surprising. The test framework shouldn't depend on the material library. At least not at this low level.

Reply:
It's purely there for the docs. Some of the docs mention `MaterialApp`, but it wasn't navigable because we didn't reference the appropriate library.

Reply:
Pretty sure that was the case for the other two new references as well.

Reply:
Oh, that's a known issue that in the IDE some doc references don't resolve because of missing imports. They do resolve once the docs are actually generated for our website. As a general rule, we do not add extra imports just to make the IDE resolution of doc references happy as these imports can have other unintended side effects.

You could upvote https://github.com/dart-lang/sdk/issues/50702, which is a proposal to fix this.

Reply:
Good to know! I'll remove this and possibly the other imports if they were just for docs as well. Thanks!

Reply:
Done.